docs/marionette.region.md

```diff
@@ -261,6 +262,26 @@ mainRegion.hasView() // true
 If you show a view in a region with an existing view, Marionette will
 [remove the existing View](#emptying-a-region) before showing the new one.
 
+## Showing a Template
+
+You can show a template or a string directly into a region. Additionally you can pass an object literal containing a template and any other view options. Under the hood a `Marionette.View` is instantiated using the template.
+
+```javascript
+var myView = new MyView();
+
+var template = _.template('This is the <%= section %> page');
```

Comment:
oh good catch.. I reformat my underscore template to use `{{}}` and `{{{}}}` so I don't have any clue what the normal is :-D